modules/mono/editor/bindings_generator.cpp

```diff
@@ -3224,6 +3265,15 @@ bool BindingsGenerator::_populate_object_type_interfaces() {
 			}
 		}
 
+		// Add compat methods that don't conflict with other methods in the type.
+		for (const MethodInterface &imethod : compat_methods) {
+			if (_method_has_conflicting_signature(imethod, itype)) {
+				WARN_PRINT("Method '" + imethod.name + "' conflicts with an already existing method in type '" + itype.name + "' and has been ignored.");
```

Comment:
This warning is noisy and there's nothing we can do about the warning anyway, so maybe it's better to remove it, or at least only print it with `--verbose`. I just wanted to make sure we were aware of ignored methods, but maybe it's not that important.